### PR TITLE
Install Eslint

### PR DESCRIPTION
**Pull request by ahmedalmi95**
install eslint 
solve eslint

### CODE REVIEW

**Review by kannanhassouna — APPROVED** [review]

**Comment by kannanhassouna** [comment]
Be careful next time, :smile: 
but you can't merge your P.R until your teammate review it

**Comment by kannanhassouna** [comment]
for merge but you will wait your team review and merge for file structure branch

**Comment by ahmedalmi95** [comment]
ok, don't worry -_*

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const app = require('./app');
+
```
Comment (Fatmasiam) [inline src/index.js:2]: I think, that's better  
```javascript
const port = app.get('port');
app.listen( port, () => {
  console.log(`the server running on port ${port}`);
});
```

**Review by Fatmasiam — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (kannanhassouna) [reply src/index.js:2]: i'm with you

**Review by Angham116 — APPROVED** [review]

**Review by Anies12 — APPROVED** [review]

**Pull request merged**